README.md

```diff
@@ -16,6 +17,13 @@ BenchBase (formerly [OLTPBench](https://github.com/oltpbenchmark/oltpbench/)) is
 
 ---
 
+## Prerequisites
+
+Java 17 needs to be installed on the system in order to run the benchbase commands.
+
+It can be downloaded directly from the [official download page.](https://www.oracle.com/java/technologies/downloads/#java17)
```

Comment:
This isn't true if you use the `docker` version.

Perhaps split this section out into the following subsections and add it there.
- Containerized
- Non-Containerized